pyartcd/pyartcd/pipelines/prepare_release.py

```diff
@@ -268,6 +269,14 @@ async def run(self):
             except CalledProcessError as ex:
                 _LOGGER.warning(f"Unable to move {impetus} advisory {advisory} to QE: {ex}")
 
+        _LOGGER.info("Check failed greenwave tests ...")
+        self._slack_client.bind_channel(self.release_name)
+        for advisory in [advisories["image"], advisories["extras"], advisories["metadata"]]:
+            test_result = get_advisory_greenwave(advisory)
+            if test_result != []:
+                _LOGGER.warning(f"Some greenwave tests on {advisory} failed with {test_result}")
```

Comment:
Can we display the number of failing tests here

Reply:
data.page.result_count should have this value but I think to know the number doesn't provide much value we need to check anyway